src/demo/previewer.js

```diff
@@ -87,6 +93,11 @@ class Previewer extends Component {
     }
   }))
 
+  onIFrameLoad = () => {
+  // Transfer port2 to the iframe
```

Comment:
Comment indentation is wrong - ESLint and Prettrier should take care of such formatting...